ibutsu-server-openshift.yaml

```diff
@@ -262,6 +106,88 @@ objects:
           namespace: ${NAMESPACE}
       type: ImageChange
     - type: ConfigChange
+# -----------------------------------------------
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: ibutsu-frontend
+    annotations:
+      description: "The frontend of Ibutsu server"
+      openshift.io/image.insecureRepository: "true"
+  spec:
+    lookupPolicy:
+      local: true
+# -----------------------------------------------
+- kind: Service
+  apiVersion: v1
+  metadata:
+    labels:
+      app: ${APP_NAME}
+    name: ibutsu-frontend
+    namespace: ${NAMESPACE}
+  spec:
+    ports:
+    - port: 8080
+      targetPort: 8080
+    selector:
+      deploymentConfig: ibutsu-frontend
+# -----------------------------------------------
+- kind: Route
+  apiVersion: v1
+  metadata:
+    labels:
+      app: ${APP_NAME}
+    name: ibutsu-frontend
+    namespace: ${NAMESPACE}
+    annotations:
+      description: "A route to the frontend"
+  spec:
+    host: ${FRONTEND_ROUTE}
+    to:
+      kind: Service
+      name: ibutsu-frontend
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+# ===============================================
+# Backend
+# ===============================================
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      app: ${APP_NAME}
+    name: ibutsu-backend
+    namespace: ${NAMESPACE}
+  spec:
+    source:
+      type: Git
+      git:
+        uri: ${IBUTSU_REPO_URL}
+        ref: ${IBUTSU_REPO_BRANCH}
+      contextDir: backend
+    strategy:
+      type: Source
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          name: python:3.6
```

Comment:
possible to use python:3.7 or 3.8?

Reply:
Hrm, let me see what's available in OCPv4

Reply:
OCPv4 only has 3.6 in the catalog. We can try 3.8 and see if it pulls it in, but I don't know that it will.